ci/scripts/build-presto-package.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
Sorry. Forgot to mention: copyright and license header needed here.

Reply:
fix